### PR TITLE
:seedling: Update CLI export with TagCategories

### PR DESCRIPTION
**Pull request by aufi**
Updating CLI tool to use TagCategories instead of TagTypes. This update doesn't cover all new resources and Hub API changes - only fixes original export/import actions to work with current Hub API.

### CODE REVIEW

**Review by jortel — APPROVED** [review]
LGTM

File: hack/tool/tackle [hunk 1]
```diff
@@ -709,6 +708,24 @@ class TackleTool:
...
+        # Compatibility checks
+        # TagCategories on Hub API
+        if apiJSON(self.tackle2Url + "/hub/tagcategories", self.tackle2Token, ignoreErrors=True) is None:
+            print("ERROR: The API doesn't know TagCategories, use older version of this tool (before mta-6.1 branch).")
+            exit(1)
+
+        # TagCategories in the data directory
+        if not os.path.exists(os.path.join(self.dataDir, 'tagcategories.json')) and os.path.exists(os.path.join(self.dataDir, 'tagtypes.json')):
+            print("ERROR: The dump comes from old version using TagTypes instead of TagCategories. Update the dump first.")
+            print("Recommended steps to update the dump: (make a backup copy of the data directory first)")
```
Comment (aufi) [inline hack/tool/tackle:720]: The tackle CLI doesn't support _update_ of exported dump to newer Hub API versions, so adding (locally tested) commands to update e.g. dump from Tackle 1.2 to be importable to current version.

**Comment by aufi** [comment]
Related to https://github.com/konveyor/tackle2-hub/issues/334

File: hack/tool/tackle [hunk 2]
```diff
@@ -709,6 +708,24 @@ class TackleTool:
             apiJSON(self.tackle2Url + tackle2path("assessments/%d" % assmnt2['id']), self.tackle2Token, data=assmnt2, method='PATCH', ignoreErrors=ignoreErrors)
 
     def preImportCheck(self):
+        # Compatibility checks
+        # TagCategories on Hub API
+        if apiJSON(self.tackle2Url + "/hub/tagcategories", self.tackle2Token, ignoreErrors=True) is None:
+            print("ERROR: The API doesn't know TagCategories, use older version of this tool (before mta-6.1 branch).")
```
Comment (jortel) [inline hack/tool/tackle:714]: I don't think we should mention MTA upstream.

**Review by jortel — APPROVED** [review]
LGTM

**Pull request merged**